yt/visualization/plot_window.py

```diff
@@ -1549,7 +1549,7 @@ def _validate_init_args(*, normal, axis, fields) -> None:
                 raise TypeError("Received incompatible arguments 'axis' and 'normal'")
             normal = axis
 
-        if (normal, fields) == (None, None):
+        if normal is fields is None:
```

Comment:
My preference is to spell this out as `if normal is None and fields is None` but this is OK.